src/test/app/NFToken_test.cpp

```diff
@@ -2957,15 +3015,32 @@ class NFToken_test : public beast::unit_test::suite
                 token::owner(minter),
                 token::destination(broker));
 
-            // Cannot broker offers when the sell destination is not the buyer
-            // or the broker.
-            env(token::brokerOffers(
-                    issuer, offerBuyerToBroker, offerMinterToBroker),
-                ter(tecNFTOKEN_BUY_SELL_MISMATCH));
-            env.close();
-            BEAST_EXPECT(ownerCount(env, issuer) == 0);
-            BEAST_EXPECT(ownerCount(env, minter) == 2);
-            BEAST_EXPECT(ownerCount(env, buyer) == 1);
+            // fixNFTokenBrokerAccept: Disabled
+            if (!features[fixNFTokenBrokerAccept])
+            {
+                // Cannot broker offers when the sell destination is not the
+                // buyer or the broker.
+                env(token::brokerOffers(
+                        issuer, offerBuyerToBroker, offerMinterToBroker),
+                    ter(tecNFTOKEN_BUY_SELL_MISMATCH));
+                env.close();
+                BEAST_EXPECT(ownerCount(env, issuer) == 0);
+                BEAST_EXPECT(ownerCount(env, minter) == 2);
+                BEAST_EXPECT(ownerCount(env, buyer) == 1);
+            }
+            else
+            // fixNFTokenBrokerAccept: Enabled
+            {
+                // Cannot broker offers when the sell destination is not the
+                // buyer
+                env(token::brokerOffers(
+                        issuer, offerBuyerToBroker, offerMinterToBroker),
+                    ter(tecNO_PERMISSION));
+                env.close();
+                BEAST_EXPECT(ownerCount(env, issuer) == 0);
+                BEAST_EXPECT(ownerCount(env, minter) == 2);
+                BEAST_EXPECT(ownerCount(env, buyer) == 1);
+            }
```

Comment:
I noticed that the only difference between the contents of the `if` and `else` branches is the `tec` code. Consider reducing the duplication by replacing the if/else with something like this:
```
            {
                // Cannot broker offers when the sell destination is not the
                // buyer or the broker.
                TER const expectTer = features[fixNFTokenBrokerAccept]
                    ? tecNO_PERMISSION : tecNFTOKEN_BUY_SELL_MISMATCH;
                env(token::brokerOffers(
                        issuer, offerBuyerToBroker, offerMinterToBroker),
                    ter(expectTer));
                env.close();
                BEAST_EXPECT(ownerCount(env, issuer) == 0);
                BEAST_EXPECT(ownerCount(env, minter) == 2);
                BEAST_EXPECT(ownerCount(env, buyer) == 1);
            }
```